docs/reference/commandline/swarm_join_token.md

```diff
@@ -11,7 +11,7 @@ parent = "smn_cli"
 # swarm join-token
 
 ```markdown
-Usage:	docker swarm join-token [--rotate] (worker|manager)
+Usage:	docker swarm join-token [OPTIONS] (worker|manager)
```

Comment:
Alright let's go for consistency with the other commands

@lixiaobing10051267 can you change the "usage" output of the actual command, so that it outputs `[OPTIONS]` instead of `[-q] [--rotate]`? You can find that part of the code here: https://github.com/docker/docker/blob/3d28cdd2f5e91e042848321de4cc4dff6735d84d/api/client/swarm/join_token.go#L19